### ISSUE TITLE
Eldar Craftworlds - Codex (2015) - update to IA11SE Doom of Mymeara

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/wh40k-7th-edition#1727]
**File:** Eldar Craftworlds - Codex (2015).catz

**BattleScribe version:** 1.15.07

**Platform:** Windows

**Dropbox:** No

**Description:** incomplete rules. no formations. no pale court. missing  new wraithknight variants. missing lord of war options.

**Comment by efalsken** [comment]
yes, Eldar are next on my list to address.

**Comment by zurran** [comment]
Eta?

**Comment by efalsken** [comment]
I'm working on Craftworlds data file now. It's a complete rewrite to ensure the best possible quality and consistency. It will be done probably next weekend, certainly by the end of the month.

**Comment by zurran** [comment]
ok kewl i was just curious

**Comment by josephking79** [comment]
How is this coming along?

**Comment by Aetherion** [comment]
I don't mean to hurry you, but what is the status on this?

**Comment by efalsken** [comment]
I've got every profile, gear, and special rules updated. The simpler units are all done, and now I'm working on the formations.

**Comment by sparkyinbozo** [comment]
Just was curious if it's been uploaded in parts or still waiting on en masse? Was curious about using a new wraithknight in a game.

**Comment by ghost** [comment]
Is help wanted with this?

**Comment by efalsken** [comment]
Nope. thanks. I've been pretty sick this week and life keeps getting in the
way. but I should have most all weekend to work on it.

On Fri, Feb 19, 2016 at 3:40 PM, capitaladot notifications@github.com
wrote:

> Is help wanted with this?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/BSData/wh40k/issues/1727#issuecomment-186454456.

## 

Eric Falsken
eric@everylittlething.net

"Expect everything, and anything seems nothing.
Expect nothing, and anything seems everything."
– Samuel Hazo

**Comment by ghost** [comment]
Progress? Could you merge an early branch and let some of us on the project start with QA?

**Comment by sparkyinbozo** [comment]
Not to pressure, but any word here?

**Comment by Jbat360** [comment]
Yeah I can't wait for this one!

**Comment by Schterve** [comment]
Just wanted to say that I'm super looking forward to this update. I appreciate all the work you've put into this tool, so thanks man!

**Comment by orlan89** [comment]
Is there a process got reassigning this ticket since it seems to have been shelved?

**Comment by efalsken** [comment]
Not shelved. We've been talking about it in Gitter. Feel free to download the latest preview from my private branch. More progress this weekend.

**Comment by orlan89** [comment]
Oh, that's wonderful. Thanks!

**Comment by efalsken** [comment]
you know ... life.... sorry for the delays.

**Comment by josephking79** [comment]
How can I download the latest preview?

**Comment by efalsken** [comment]
If you don't know how to use Git, then it's really not worth the trouble. You'll just mess yourself up.

**Comment by orlan89** [comment]
@efalsken, I downloaded your repo and am going to give it a test. I've already noticed that Scat Bikes only cost 17 instead of 27. If I find additional items like this, how would you like me to relay that to you? 

Thanks again for what you're doing!

**Comment by efalsken** [comment]
Thanks. Go ahead and leave the feedback here for now. Once I issue the pull request and start the merge, then we can track issues there separately.

**Comment by orlan89** [comment]
Okay, will do

**Comment by TheRedViking** [comment]
Hey I figured as I am excited for this release I might as well help with checking for issues I have attached a text file with what I noticed to be missing. First though, Thank you very much for your work on updating my favorite army.
[EldarRulesCheck.txt](https://github.com/BSData/wh40k/files/173039/EldarRulesCheck.txt)

**Comment by efalsken** [comment]
@TheRedViking: Thanks for this. Great detective work! I will get this integrated ASAP and a new update tonight.

**Comment by TheRedViking** [comment]
As well I did not see any Forgeworld units will these be added later?

**Comment by sparkyinbozo** [comment]
Wasn't adding FW units a big focus of this fix...? If not, should we open up another report/have someone else work on it?

**Comment by TheRedViking** [comment]
Maybe we should give efalsken a chance to reply, he may want to get the core codex working before adding the forgeworld stuff in.

**Comment by orlan89** [comment]
@sparkyinbozo I read the gitter chat regarding this. Efalsken was redoing the entire Craftworlds codex first. Once that is good to go, he will move on to the FW units.

**Comment by efalsken** [comment]
Thanks @orlan89. Yes, the previous catalog had 3 different copies of some rules, no (or very few) rule citations, etc. I could have smashed the new units into it with a minimum of effort, but I felt that Eldar deserved the attention. After I built the Tau data files, I really didn't like even playing with the current Eldar data file. This one will be much simpler and cleaner to maintain.

**Comment by sparkyinbozo** [comment]
Ahh, that makes a lot of sense.  Thank you for all of the hard work!

**Comment by mystycalchemy** [comment]
[Eldar Rules Errors.txt](https://github.com/BSData/wh40k/files/178890/Eldar.Rules.Errors.txt)

Found these errors as well, thanks for your hard work @efalsken

**Comment by Threeshades** [comment]
Hey, I sat down one night and just put in all the current Forge World stuff from Doom of Mymeara just because I needed a Wraithseer. I haven't submitted it to the repository. Maybe this will help you.
[Eldar Craftworlds - Codex (2015).zip](https://github.com/BSData/wh40k/files/179877/Eldar.Craftworlds.-.Codex.2015.zip)

**Comment by efalsken** [comment]
@mystycalchemy Thanks! I've made these changes and pushed the update (again to my repo).

**Comment by LordPathos** [comment]
I'm new to the site so I haven't figured everything out yet, but is the work you're doing to add the Doom of Mymeara stuff for the iphone app only or also the other platforms? Specifically for windows.

**Comment by efalsken** [comment]
@LordPathos: Battlescribe catalogs can be used on both platforms without any problems. You can even use Dropbox to sync your data and rosters from your phone to your desktop for easy printing. I recommend the Battlescribe Facebook group or Reddit to get more help with how to use Battlescribe.

**Comment by LordPathos** [comment]
I think it's kind of silly that on a site for community based stuff, you can't send messages to each other. Or if you can, I haven't figured out how. But all I want is to get the Doom of Mymeara stuff so I can make my Pale Court list. How do I just grab your catelog for my Battlescribe?

**Comment by amis92** [comment]
@LordPathos for chitchat we use gitter: https://gitter.im/BSData/wh40k (also provides PM)

**Comment by LordPathos** [comment]
Actually now that I'm looking at the categories efalsken has, where is the one with the Mymeara stuff? The current Eldar catergory hasnt been updated for 4 months.

**Comment by efalsken** [comment]
@LordPathos the whole point of this GitHub issue is to track the progress on the Mymeara update. It's not ready for you to use yet. It's especially difficult to get the work-in-progress if you are unfamiliar with GitHub tooling. Beyond that, even if you could get my WIP files, Mymeara updates are still being entered and Pale Courts haven't even been started yet. Your best solution, if you're desperate for the Mymeara units now, is to use the file that Threeshades posted a screen or two up. Just replace the Eldar file in your battlescribe data folder. This will be impossible to do on a mobile device, so you're going to need to use the Desktop edition. This will also cause problems when you want to resync with the public release once my final version is ready.

**Comment by efalsken** [comment]
@Everyone: My branch has the latest work on the Mymeara-Craftworlds updates and Pale Courts. Please review it. If it looks good, then I will replace the current Eldar Craftworlds catalog and start the Pull Request. I'd like to give this a good beatdown before pulling the trigger. The Pale Courts' Forgotten Remnants rules are anything but easy to implement.

**Comment by mystycalchemy** [comment]
Found a few bugs with the new push.
- Bel-Annath has an option to add more rune armors. 
- Wraithblades can mix and match their weapons (But im actually more of a fan of how it works compared to the old catalogue)
- Shadow Spectres can add multiple exarchs to the squad.
- Warp Hunters start with 0 members in the squad.
  -Skathach Wraithknight can only take one of each secondary weapon (One scatter laser, one shuriken cannon, etc.)

Thats all I've found so far. Thanks again @efalsken 
EDIT: I Also found this: 
- If building a CAD or Unbound army, you can take a Crimson Death formation as a Fortification choice.
- At the top of the categories, theres a (No Category) with no options inside it.
- The Craftworld Warhost Detatchment is missing the Wraith Constructs choice under Auxiliary

**Comment by efalsken** [comment]
@mystycalchemy thanks. Fixed and pushed. The `(No Category)` is a Battlescribe artifact since there are possible choices there depending on who/what you bring; Bel-Annath enables a unit of Fire Dragons. I'm thinking that they may be better under Bel-Annath, but they're not supposed to be part of his formation.

**Comment by mystycalchemy** [comment]
Couple more things noticed
- Wraithblade Weapons cost 30 points no matter which you choose, should be free AFAIK.
- Windrider host is misspelled (Spelled as Winrider host in the warhost detatchment.)
- Also in the Windrider host, the Warlock Conclave has two options, Singing Spear or Witchblade, however when you add Spear or Blade Warlocks, you have to click on them and add a Spear or Blade for it to appear under their wargear.

**Comment by efalsken** [comment]
Thanks again. Fixed!

**Comment by gurtyel** [comment]
I noticed a bug:

The Aspect lord-shrine pale court's trait doesnt supress the need to add 3 guardians as a core choice as itself, suppossedly choosing 3 aspect warriors units from the same aspect should allow me to field not a single guardian unit if i so desire, it becomes (0-3). 

But in the push you submitted, it prompts an error in the list's log saying that i still need to add 3 guardian units in my core choice. Below this, it shows the following text:

set minselections to 0 if pale courts battlehost has equal to 1 selections of graveyard of dreams AND, Pale courts battlehost has equal to 1 selections of aspect lord-shrine. 

I believe the relevant keyword in this sentence shoulb be OR instead of AND as both traits supress the need to add guardians to the core choice, you dont need to have selected both traits to enjoy this benefit you just need one of those traits.

**Comment by efalsken** [comment]
That's one of the most troublesome rules in the Pale Courts.... I will try to do another push for that one in a little bit.

**Comment by efalsken** [comment]
Fixed Pale Courts - Aspect Lord-Shrine

**Comment by josephking79** [comment]
Pale Courts - Crossroads of Eternity permits 0-1 Harlequin Troupe and/or 0-1 Corsair Reader band, but there doesn't seem to be any way of adding them

**Comment by amis92** [comment]
To be honest, the exact reason for existence Pull Requests is what is happening now. So, they are meant to be a place for discussion of new additions/changes before merging them. We can see then updates as part of discussion and have easier access through desktop GitHub client. :)

**Comment by efalsken** [comment]
Hmm. ok. Will do the PR now.

**Comment by LordPathos** [comment]
Hello again,

I didn't mean to sound super impatient the other day. I didn't realize how much work has to go into updating this stuff. So I will try to help the best I can. I have the Mymeara book, so I can reference rules if you need. For instance, the Graveyard of Dreams trait lets you take 0-3 guardians, but you have to take 3 units of Wraithblades. I saw the isse with the Aspect Shrine, so I just wanted to let you know another trait does the same thing to the Guardians.

**Comment by Aetherion** [comment]
Just one last question, is it automatically synced with mobile yet?

**Comment by efalsken** [comment]
@Aetherion Everyone should see it now. If you do not, press the "refresh" button on the Data Files screen.

**Comment by sparkyinbozo** [comment]
Wasn't the Scathach Wrairhknight's shunt supposed to allow 18" movement, then scatter?  There was a FW e-mail floating around that said 12" was an error. Link here:

https://www.reddit.com/r/Warhammer40k/comments/4412z7/doom_of_mymeara_answers_from_forgeworld/

**Comment by efalsken** [comment]
@sparkyinbozo this issue is closed. if you found a bug, please open a new ticket. Unfortunately, BSData team goes with rules-as-printed in all but the most obvious mistakes. Emails aren't official sources of rules.

**Comment by tharsiel** [comment]
I'm new and wanted to download the eldarfile but couldn't find it, so can anyone tell how?
